lib/ecto/adapters/postgres/connection.ex

```diff
@@ -626,6 +633,25 @@ if Code.ensure_loaded?(Postgrex) do
       end
     end
 
+    defp comment_on(_database_object, _name, nil), do:  ""
+    defp comment_on(:column, {table_name, column_name}, comment) do
+      column_name = quote_table(table_name, column_name)
+      "; COMMENT ON COLUMN #{column_name} IS #{single_quote(comment)}"
+    end
+
+    defp comment_on(database_object, name, comment) do
```

Comment:
Instead of doing Atom.to_string |> String.upcase, we should explicitly list everything we support. :)